baselines/dom.generated.d.ts

```diff
@@ -16310,7 +16310,23 @@ declare var MessageEvent: {
     new<T>(type: string, eventInitDict?: MessageEventInit<T>): MessageEvent<T>;
 };
 
-interface MessagePortEventMap {
+interface MessageEventTargetEventMap {
+    "message": Event;
+    "messageerror": Event;
+}
+
+interface MessageEventTarget {
+    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/message_event) */
```

Comment:
The link is suboptimal in this very file. Optimal would be
https://developer.mozilla.org/en-US/docs/Web/API/Window/message_event

Reply:
Nothing is optimal actually as MessageEventTarget can be implemented by multiple interfaces. And Window doesn't implement it, that one is coming from WindowEventHandlers. Using worker on window lib is indeed weird though.